Makefile

```diff
@@ -7,8 +7,7 @@ all: virtualenv install
 
 virtualenv:
 	@if [ ! -d "venv" ]; then \
-		$(PYTHON_BIN) -m pip install virtualenv --user; \
```

Comment:
I don't use virtualenv. What does it provide that the built-in venv would not?

Reply:
It's just personal preference. Many features of `virtualenv` were baked in to `venv` back in 3.3, but `virtualenv` is still maintained as its faster and usually several core features ahead of the built-in lib.

https://virtualenv.pypa.io/en/latest/

Reply:
I suggested the change because of this error:
```error
× This environment is externally managed
╰─> To install Python packages system-wide, try 'pacman -S
    python-xyz', where xyz is the package you are trying to
    install.
...
```